src/analysis/explorerHomeView.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import {CredGraph} from "../core/credrank/credGraph";
+import {type TimestampMs} from "../util/timestamp";
+import {type Interval} from "../core/interval";
+import * as Grain from "../core/ledger/grain";
+
+export type DataOverTime = $ReadOnlyArray<number>;
+
+export type UserRow = {|
+  +name: string,
+  +credEarnedInTimeRange: number,
+  +grainHarvestedInTimeRange: Grain.Grain,
+  +grainBalance: Grain.Grain,
+  +allTimeCred: DataOverTime,
+  +category: UserCategory,
+|};
+
+export type UserCategory = "participant" | "bot" | "project" | "organization";
+
+export type ExplorerHomeView = {|
+  +credEarned: number,
+  +grainHarvested: Grain.Grain,
+  +participantCount: number, //Question: what is this supposed to capture? Is it supposed to change depending on the time range selected?
```

Comment:
We have this idea of defining "active participants" per https://github.com/sourcecred/sourcecred/issues/2244

It will change over time because whether you are considered "active" depends on your level of activity in the chosen period

Reply:
right, but for the first pass we said totalParticipants, right? does that just mean a static value that doesn't change with time selection?